packages/plugin-autocapture-browser/src/autocapture/track-dead-click.ts

```diff
@@ -27,55 +20,60 @@ export function trackDeadClick({
   getEventProperties: (actionType: ActionType, element: Element) => Record<string, any>;
   shouldTrackDeadClick: shouldTrackEvent;
 }) {
-  const { clickObservable, mutationObservable, navigateObservable } = allObservables;
-
-  const filteredClickObservable = clickObservable.pipe(
-    filter(filterOutNonTrackableEvents),
-    filter((clickEvent) => {
-      // Only track change on elements that should be tracked
-      return shouldTrackDeadClick('click', clickEvent.closestTrackedAncestor);
-    }),
-    filter((clickEvent) => {
-      const target = clickEvent.event.target as Element;
-      return target.closest('a[target="_blank"]') === null;
-    }),
-  );
+  const { clickObservableZen, mutationObservableZen, navigateObservableZen } = allObservables;
 
-  const changeObservables: Array<
-    AllWindowObservables[ObservablesEnum.MutationObservable] | AllWindowObservables[ObservablesEnum.NavigateObservable]
-  > = [mutationObservable];
-  if (navigateObservable) {
-    changeObservables.push(navigateObservable);
+  /* istanbul ignore if */
+  if (!clickObservableZen || !mutationObservableZen) {
+    return;
   }
-  const mutationOrNavigate = merge(...changeObservables);
 
-  const actionClicks = filteredClickObservable.pipe(
-    mergeMap((click) => {
-      // Create a timer that emits after 500ms
-      let timeoutId: ReturnType<typeof setTimeout>;
-      const timer = new Observable<typeof click>((subscriber: Subscriber<typeof click>) => {
-        timeoutId = setTimeout(() => subscriber.next(click), DEAD_CLICK_TIMEOUT);
+  const filteredClickObservable = clickObservableZen.filter((click) => {
+    return (
+      filterOutNonTrackableEvents(click) &&
+      shouldTrackDeadClick('click', click.closestTrackedAncestor) &&
+      click.event.target instanceof Element &&
+      click.event.target.closest('a[target="_blank"]') === null
+    );
+  });
+
+  /* istanbul ignore next */
+  const changeObservables = navigateObservableZen
+    ? merge(mutationObservableZen, navigateObservableZen)
+    : mutationObservableZen;
+
+  const clicksAndChangeObservable = merge(filteredClickObservable, changeObservables);
 
-        return () => {
-          clearTimeout(timeoutId);
-        };
-      });
+  let deadClickTimer: NodeJS.Timeout | null = null;
```

Comment:
Type seems incorrect since this is not a NodeJs env

NodeJS.Timeout -> ReturnType<setTimeout>